src/util/map.js

```diff
@@ -122,3 +122,26 @@ export function mapEntries<K, V, InK, InV>(
   }
   return result;
 }
+
+/**
```

Comment:
Note that the analogous operation on `Graph` is called `merge`, and also
this function has a different API from `Array.prototype.concat` (and a
better one). Weak preference toward `merge` for those reasons, and also
because `concat` suggests a notion of order that maps don’t\* have.